README.md

```diff
@@ -749,7 +750,8 @@ Also, unlike those above, which requires runtime dependancy, some pure-Nim libra
 
 ### Build Systems / Package Management
 
-- [ChooseNim](https://github.com/dom96/choosenim) - Installing and switching between Nim versions (à la rustup, pyenv).
+- [ChooseNim](https://github.com/nim-lang/choosenim) - Installing and switching between Nim versions (à la rustup, pyenv).
+- [GrabNim](https://codeberg.org/janAkali/grabnim) - Install and switch between multiple Nim compiler versions. (Alternative to ChooseNim).
```

Comment:
Isn't this an old change in https://github.com/ringabout/awesome-nim/pull/104? I don't know why Github have shown them here